docs/src/pages/[platform]/components/togglebutton/examples/ToggleButtonThemeExample.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  ToggleButton,
+  Flex,
+  ThemeProvider,
+  Theme,
+} from '@aws-amplify/ui-react';
+
+const theme: Theme = {
+  name: 'toggleButton-theme',
+  tokens: {
+    components: {
+      togglebutton: {
+        borderColor: { value: '{colors.brand.secondary.80}' },
```

Comment:
The border color and text color may need to adjust a bit
![Screen Shot 2022-06-16 at 3 45 55 PM](https://user-images.githubusercontent.com/40295569/174190643-99d8f041-9643-43df-9813-194c337c6f27.png)